src/lib/profiles/data-management/Current/TraitData.h

```diff
@@ -1,5 +1,6 @@
 /*
  *
+ *    Copyright (c) 2018 Google LLC.
```

Comment:
Is that the new copyright message that we need to incorporate in all new files?
Don't mention Nest Labs any more?

Reply:
Confirmed.